daphne_worker/src/durable/report_store.rs

```diff
@@ -215,8 +215,18 @@ impl DurableObject for ReportStore {
             }
 
             (DURABLE_REPORT_STORE_PUT_PROCESSED, Method::Post) => {
-                let nonce_hex: String = req.json().await?;
-                Response::from_json(&self.checked_process(&nonce_hex).await?)
+                let nonce_hex_set: Vec<String> = req.json().await?;
+                let mut futures = Vec::new();
+                for nonce_hex in nonce_hex_set.iter() {
+                    futures.push((nonce_hex.clone(), self.checked_process(nonce_hex)));
+                }
+                let mut res = Vec::with_capacity(futures.len());
+                for (nonce_hex, future) in futures.into_iter() {
```

Comment:
Is it possible to benchmark this loop? It's not clear to me if it's the communication cost to the DO that dominates, or the cost for processing each nonce that dominates.

Reply:
This PR adds the duration of each request to logs. I can confirm on a live deployment that coalescing significantly improves runtime.

Reply:
Sorry, I'm asking about benchmarking what's happening within each request. With this change, each request processes a set of nonces. How long does processing each nonce take?

Reply:
OHhh got it. Yeah we don't have that yet.